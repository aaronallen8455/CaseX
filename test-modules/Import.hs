module Import
  ( Foo(..)
  ) where

data Foo = Foo | Bar | Baz
