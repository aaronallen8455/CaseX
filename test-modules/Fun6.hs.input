import Import

main :: IO ()
main = pure ()

test :: Maybe Foo -> Bool
test (Just SPLIT) = True
test Nothing = False
