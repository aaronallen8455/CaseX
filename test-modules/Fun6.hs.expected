import Import

main :: IO ()
main = pure ()

test :: Maybe Foo -> Bool
test (Just Foo) = True
test (Just Bar) = True
test (Just Baz) = True
test Nothing = False
